Add how to change models (#335)
sourcegraph/docs: docs/cody/clients/install-vscode.mdx

  <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/Chat_%20Redesigned%20chat%20editing.mp4" type="video/mp4" />
</video>

### Changing model used
NOTE: You need to be a Cody Pro user to have multi-model selection capability. This is not available for Cody Free or Cody Enterprise users

For Chat:
1. Open a new chat window (Opt + L)

> 1. Click on the default chat name (at the time of writing this is Claude 3 Sonnet)

Reviewer: ```suggestion
1. Click on the model selector (which by default indicates Claude 3 Sonnet)
```

1. See selection of models and click the model you desire. This model will now be the default model going forward on any new chats.

For Edit:
1. On any file select some code and do a right-click

> 1. Select Cody->Edit Code (optionally you can do this with Opt + K)

Reviewer: ```suggestion
1. Select Cody->Edit Code (optionally you can do this with Opt+K/Alt+K)
```
